holidayextras/ui-toolkit
# All children are being displayed as table.

#### What does this PR do? (please provide any background)
We noticed that all child elements are having `display:table-cell` added to, grand children and great grand children.
This PR ensures only direct children are affected and cleans up the specificity a bit.

#### What tests does this PR have?
None


#### How can this be tested?
- Run `npm run docs` and edit the basket item molecules code on the docs.
- Try adding a `<span>` in the desciption of the basket item.
- You should see no change to the displaying of the information on the page.
- Now visit http://www.ui-toolkit.com and try the same thing. It makes text bunch up

#### Screenshots / Screencast
** Before **
<img width="802" alt="screen shot 2015-11-25 at 11 17 08" src="https://cloud.githubusercontent.com/assets/778942/11395951/1df34d48-9366-11e5-8fbb-4191b99bea81.png">

** After **
<img width="808" alt="screen shot 2015-11-25 at 11 18 15" src="https://cloud.githubusercontent.com/assets/778942/11395962/3f8e29c8-9366-11e5-9b6d-d6d6a66992d6.png">


#### What gif best describes how you feel about this work?
![Dog Jump](https://media.giphy.com/media/COyNr59BTUKek/giphy.gif)


---


#### Reviewers

**Review 1**

**Review 2** \*

**Review 3** _(optional)_

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru
